src/main/java/se/sundsvall/invoicesender/integration/db/entity/ItemEntity.java

```diff
@@ -40,6 +57,71 @@ public void setId(final Integer id) {
 		this.id = id;
 	}
 
+	public BatchEntity getBatch() {
+		return batch;
+	}
+
+	public void setBatch(BatchEntity batch) {
+		this.batch = batch;
+	}
+
+	public ItemEntity withBatch(BatchEntity batch) {
+		this.batch = batch;
+		return this;
+	}
+
+	public String getRecipientPartyId() {
+		return recipientPartyId;
+	}
+
+	public void setRecipientPartyId(String recipientPartyId) {
+		this.recipientPartyId = recipientPartyId;
+	}
+
+	public ItemEntity withRecipientPartyId(String recipientPartyId) {
+		this.recipientPartyId = recipientPartyId;
+		return this;
+	}
+
+	public String getRecipientLegalId() {
+		return recipientLegalId;
+	}
+
+	public void setRecipientLegalId(String recipientLegalId) {
+		this.recipientLegalId = recipientLegalId;
+	}
+
+	public ItemEntity withRecipientLegalId(String recipientLegalId) {
+		this.recipientLegalId = recipientLegalId;
+		return this;
+	}
+
+	public Metadata getMetadata() {
+		return metadata;
+	}
+
+	public void setMetadata(Metadata metadata) {
+		this.metadata = metadata;
+	}
+
+	public ItemEntity withMetadata(Metadata metadata) {
+		this.metadata = metadata;
+		return this;
+	}
+
+	public ItemType getType() {
+		return type;
+	}
+
+	public void setType(ItemType type) {
+		this.type = type;
+	}
+
+	public ItemEntity withType(ItemType type) {
+		this.type = type;
+		return this;
+	}
+
```

Comment:
No ToString / Equals / Haschcode?

Reply:
None of them are used in this application.